features/support/activitypub.js

```diff
@@ -70,6 +70,28 @@ export async function findInOutbox(activity) {
     return (outbox.orderedItems || []).find((item) => item.id === activity.id);
 }
 
+export async function findNoteInOutbox(note) {
```

Comment:
This is really finding an object, right? I don't think it has to be specific to a note

Reply:
Updated tests: https://github.com/TryGhost/ActivityPub/pull/551/commits/ed7cda862e6931a298c4971328ed2316cbc4863e